ui/src/pages/job/JobPage.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2020 The Merlin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { replaceBreadcrumbs } from "@caraml-dev/ui-lib";
+import {
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingChart,
+  EuiPageTemplate,
+  EuiSpacer,
+  EuiTextAlign,
+} from "@elastic/eui";
+import React, { Fragment, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useMerlinApi } from "../../hooks/useMerlinApi";
+import mocks from "../../mocks";
+import JobConfig from "./components/JobConfig";
+import { JobDetailTabNavigation } from "./components/JobDetailTabNavigation";
+import JobErrorMessage from "./components/JobErrorMessage";
+import { JobInfoPanel } from "./components/JobInfoPanel";
+import JobLog from "./components/JobLog";
+import { JobRunPanel } from "./components/JobRunPanel";
+import { generateBreadcrumbs } from "./utils/breadcrumbs";
+
+const JobPage = () => {
+  const { projectId, modelId, versionId, jobId, "*": section } = useParams();
+
+  const [{ data: project, isLoaded: projectLoaded }] = useMerlinApi(
+    `/projects/${projectId}`,
+    { mock: mocks.project },
```

Comment:
why is this named mock?

Reply:
In Merlin, we can develop the UI without actually calling the API by mocking the response data. To do that:

1. Enable `USE_MOCK_DATA` https://github.com/caraml-dev/merlin/blob/main/ui/src/config.js#L99
2. Create mock data for a given resource in mocks folder: https://github.com/caraml-dev/merlin/tree/main/ui/src/mocks
3. Pass it to useMerlinApi mock field.